.dependabot/config.yml

```diff
@@ -0,0 +1,5 @@
+version: 1
+update_configs:
+- package_manager: go:modules
+  directory: /
+  update_schedule: daily
```

Comment:
Since we have SO many repos, daily might be too frequent?
We should adjust it later. (but maybe not now)